integration-tests/tests/test_utils/trait_mod.rs

```diff
@@ -0,0 +1,347 @@
+use axelar_wasm_std::nonempty;
```

Comment:
I don't how I feel about all of the trait implementations for every contract existing in this one file. Can we move each to it's own file?

Reply:
Also, should the trait implementations live under the src folder? With the trait definition itself?

Reply:
I will move each implementation to its own contract.
About the second question, there were other options but I talked with Chris and decided to put it there. We can change of course.